boards/shields/adafruit_2_8_tft_touch_v2/doc/index.rst

```diff
@@ -0,0 +1,68 @@
+.. _adafruit_2_8_tft_touch_v2:
+
+Adafruit 2.8" TFT Touch Shield v2
+#################################
+
+Overview
+********
+
+The Adafruit 2.8" TFT Touch Shield v2 with a
+resolution of 320x240 pixels, is based on the ILI9341 controller.
+This shield comes with a resistive (STMPE610 controller)
```

Comment:
As written, is sounds like Zephyr doesn't support these displays at all. I think you're saying Zephyr supports output to the displays, but does not support touchscreen input, right?  If so, let's rewrite this as:
```
This shield comes with a resistive (STMPE610 controller)
or capacitive (FT6206 controller) touchscreen.  While the
Zephyr RTOS supports display output to these screens,
it currently does not support touchscreen input.
```